libs/server/controllers/friend/src/lib/friend.service.ts

```diff
@@ -79,19 +83,17 @@ export class FriendService {
       where: { id: data.requesteeId },
     });
 
-    await this.prismaService.user.update({
-      where: { id: data.addresseeId },
+    const notification = await this.prismaService.notification.create({
       data: {
-        notifications: {
-          create: {
-            title: 'Neue Freundschaftsanfrage',
-            description: `${requestee.username} hat dir eine Freundschaftsanfrage gesendet`,
-            type: NotificationType.FRIEND_REQUEST_RECEIVED,
-          },
-        },
+        title: 'Neue Freundschaftsanfrage',
+        description: `${requestee.username} hat dir eine Freundschaftsanfrage gesendet`,
```

Comment:
Line Breaks in da Description

Reply:
Beim toast passts glob e:
![image](https://user-images.githubusercontent.com/54234287/208477810-09f73208-ed1c-4a1d-92e4-2df27fec4f36.png)

Mit breaks würds so usluaga:
![image](https://user-images.githubusercontent.com/54234287/208478903-1c2176fc-2f56-4874-ac86-9b030c24487a.png)

Reply:
sehr nice👍, isch es aktuell so?

Reply:
yes